flaskr.py

```diff
@@ -80,4 +80,16 @@ def login():
 def logout():
 	session.pop('logged_in', None)
 	flash('You were logged out')
+	return redirect(url_for('show_entries'))
+
+@app.route('/delete=<int:entry_id>', methods=['GET','POST'])
+def delete_entry(entry_id):
+	error=None
+	if request.method == 'POST':
```

Comment:
If you define this route with `methods=['POST']`, this check can be removed. Flask will do this and make sure this route is only called with POST method.